sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 3.0.0-preview.2 (2020-08-18)
```

Comment:
This will probably generate a conflict. Feel free to move stuff that was already listed in 3.0.0-preview.2 to 3.0.0-preview.3.